RELEASE.md

```
@@ -0,0 +1,105 @@
+A GitHub Actions workflow has been setup to automate building, scanning, packaging, and releasing npm packages in the `packages/` directory.
+
+---
+
+## 🔑 **Usage**
+
+You can manually trigger the release workflow from the [GitHub Actions UI](https://github.com/xrplf/xrpl.js/actions/workflows/release.yml).
+
+### **Before triggering a release**
+
+1. Create a release branch and update the **`version`** field in `packages/<package_name>/package.json` to the intended release version.
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)